services/QuillLMS/app/queries/activity_search.rb

```diff
@@ -12,9 +12,9 @@ def self.search(flag)
     when 'alpha'
       flags = "'alpha', 'beta', 'gamma', 'production'"
     when 'beta'
-      flags = "'beta', 'gamma', 'production'"
+      flags = "'beta', 'production'"
```

Comment:
@anathomical FYI I added this change to the PR, since ActivitySearch uses a separate code path.

Reply:
👍